### PR TITLE
put each sprite in seperate folders

### PR DESCRIPTION
**Pull request by H-A-M-G-E-R**
when there's only 1 texture in a sprite it doesn't create a folder

### CODE REVIEW

**Comment by H-A-M-G-E-R** [comment]
but it breaks ImportGraphics.csx

**Comment by H-A-M-G-E-R** [comment]
i made it compatible with ImportGraphics.csx

File: UndertaleModTool/Scripts/Resource Repackers/ImportASound.csx [hunk 1]
```diff
@@ -211,7 +211,7 @@ if (dialogRet == DialogResult.OK)
             File        = Data.Strings.MakeString(fname),
             Effects     = 0,
             Volume      = 1.0F,
-            Pitch       = 1.0F,
+            Pitch       = 0,
```
Comment (VladiStep) [inline UndertaleModTool/Scripts/Resource Repackers/ImportASound.csx:214]: I think this is a redundant change that you should undo.
```suggestion
            Pitch       = 1.0F,
```

Comment (VladiStep) [reply UndertaleModTool/Scripts/Resource Repackers/ImportASound.csx:214]: You should undo this, and instead create a separate PR with only this particular change.

**Comment by Miepee** [comment]
obsoleted in favour of the above mentioned PR

**Pull request closed without merging**